.ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml

```diff
@@ -27,8 +27,6 @@ pin_run_as_build:
     max_pin: x
   expat:
     max_pin: x.x
-  libnetcdf:
-    max_pin: x.x.x
```

Comment:
@ocefpaf, was this expected?  Are we no longer pinning `libnetcdf` run as build? Or is the bot glitching?  Either one isn't good news.

Reply:
As I commented elsewhere, maybe it's fine.  https://abi-laboratory.pro/index.php?view=timeline&l=netcdf shows 100% compatibility going back many releases.

Reply:
In theory [this should](https://github.com/conda-forge/libnetcdf-feedstock/blob/099ecbb1f53e182ca874a5e50f05965b3123e47a/recipe/meta.yaml#L55) take care of that.

Reply:
Ah, thanks for the explanation.